src/Chapter1.hs

```diff
@@ -541,8 +544,8 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z = (x + y + z) - (min (min x y) z) - (max (max x y) z)
```

Comment:
Very interesting solution! 💡 
Though it could be a bit simplified. You see that all you need is to compare the absolute value of both `x` and `y`, and based on that return the result (as you do in the `then` and `else` clauses 👍🏼